src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -504,6 +504,7 @@
      (let [suggested-routes-new-data (data-store/rpc->suggested-routes data)
            suggested-routes          (-> suggested-routes-new-data
                                          first
+                                         (update :best (fn [best] (if (> (count best) 1) [] best)))
```

Comment:
I think we should add a comment here to explain this very random behaviour :)

Reply:
Added!

Reply:
What will the UI show if the multichain route is removed?

Reply:
In this case it would end to a `No routes found` case